demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,491 @@
+r"""
+
+.. _classical_kernels:
+
+Emulating classical kernels
+============================
+
+.. meta::
+    :property="og:description": Show how to approximate the Gaussian kernel.
+    :property="og:image": https://pennylane.ai/qml/_images/boring_kernel.jpg
+
+.. related::
+
+   tutorial_kernels_module Training and evaluating quantum kernels
+   tutorial_kernel_based_training Kernel-based training of quantum models with
+   scikit-learn
+   tutorial_expressivity_fourier_series Quantum models as Fourier series
+
+
+*Author: Elies Gil-Fuster (Xanadu resident). Posted: DD MMMM 2021.*
+
+In this demo we will briefly revisit some notions of kernel-based Machine
+Learning (ML) and introduce one very specific `Quantum Embedding Kernel (QEK)
+<https://pennylane.ai/qml/demos/tutorial_kernels_module.html>`_.
+We will also use this QEK to demonstrate we can approximate a large class of
+classical kernels with it, known as shift-invariant or stationary kernels.
+Both the math and the code will stay high-level throughout the explanation,
+this is not one of those very technical demos!
+So, if you feel like exploring one link between classical and quantum kernels,
+this is the place for you!
+
+|
+
+.. figure:: ../demonstrations/classical_kernels/sketch.PNG
+    :align: center
+    :width: 80%
+    :target: javascript:void(0)
+
+    The Quantum Embedding Kernel covered in this demo. 
+
+
+
+Kernel-based Machine Learning
+----------------------------------------------
+
+As we just said, in the interest of keeping the concepts at a high level we
+will not be reviewing all the notins of kernels in-depth here.
+Instead, we only need to know that there's an entire branch of ML which
+revolves around some functions we call kernels.
+If you'd still like to know more about where these functions come from, why
+they're important, and how we can use them (e.g. with PennyLane), luckily there
+are already two very nice demos that cover different aspects extensively:
+
+#. `Training and evaluating quantum kernels <https://pennylane.ai/qml/demos/tutorial_kernels_module.html>`_
+#. `Kernel-based training of quantum models with scikit-learn <https://pennylane.ai/qml/demos/tutorial_kernel_based_training.html>`_
+
+Now, a kernel for us is a real-valued function of two variables
+:math:`k(\cdot,\cdot)` from a given data domain :math:`x_1, x_2\in\mathcal{X}`.
+Further, we require a kernel to be symmetric to exchanging the variable
+positions :math:`k(x_1,x_2) = k(x_2,x_1)`.
+Finally, we will also want to enforce the kernels be positive semi-definite,
+but let's avoit getting lost in mathematical definitions, you can trust that
+all kernels featuring in this demo are positive semi-definite.
+
+If you take whichever textbook on kernel methods and search for the word
+``prominent", chances are you'll find it next to the word ``example" in a
```

Comment:
```suggestion
"prominent", chances are you'll find it next to the word "example" in a
```
This is not latex ;)

Reply:
oopsie!